src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -2101,4 +2384,191 @@ instruct vloadcon(vReg dst, immI0 src) %{
     }
   %}
   ins_pipe(pipe_slow);
+%}
+
+instruct vmask_gen_I(vRegMask dst, iRegI src) %{
+  match(Set dst (VectorMaskGen (ConvI2L src)));
+  format %{ "vmask_gen_I $dst, $src" %}
+  ins_encode %{
+    __ clear_register_v(as_VectorRegister($dst$$reg));
+    __ vsetvli(t0, $src$$Register, Assembler::e8);
+    __ vmxnor_mm(as_VectorRegister($dst$$reg), as_VectorRegister($dst$$reg), as_VectorRegister($dst$$reg));
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+instruct vmask_gen_L(vRegMask dst, iRegL src) %{
+  match(Set dst (VectorMaskGen src));
+  format %{ "vmask_gen_L $dst, $src" %}
+  ins_encode %{
+    __ clear_register_v(as_VectorRegister($dst$$reg));
+    __ vsetvli(t0, $src$$Register, Assembler::e8);
+    __ vmxnor_mm(as_VectorRegister($dst$$reg), as_VectorRegister($dst$$reg), as_VectorRegister($dst$$reg));
```

Comment:
We could introduce assembler pseudoinstructions 'vmmv.m', 'vmclr.m', 'vmset.m' and 'vmnot.m' to make the vector mask computation more readable.
```
vmmv.m vd, vs => vmand.mm vd, vs, vs # Copy mask register
vmclr.m vd => vmxor.mm vd, vd, vd # Clear mask register
vmset.m vd => vmxnor.mm vd, vd, vd # Set mask register
vmnot.m vd, vs => vmnand.mm vd, vs, vs # Invert bits
```

Reply:
Fixed.